pkgs/ffigen/lib/ffigen.dart

```diff
@@ -8,5 +8,6 @@
 library ffigen;
 
 export 'src/code_generator.dart' show Library;
```

Comment:
Let's audit the public API as well.

Maybe we should have one public `library` per use case.

* `package:ffigen/ffigen.dart` for working with `class FfiGen` and `class Config` to basically do what one did in ffigen.yaml in Dart
* `package:ffigen/programmatic_interface.dart` (name TBD) that has `Library` and `parse`. (Is anyone using that use case right now though? Maybe omit these from the public API completely atm.)

Reply:
Done